### ISSUE TITLE
[build] ZEPHYR_BASE not set when make update

### ISSUE THREAD

**Issue by haoxli** [01org/zephyr.js#719]
Clone project in a new place and update deps/, it will check **ZEPHYR_BASE**, but there is no _deps/zephyr/zephyr-env.sh_ before update successfully, the update stop.

```
$ git clone https://github.com/01org/zephyr.js 
$ cd zephyr.js
$ source zjs-env.sh
ZJS_BASE: /work/weekly/ww07/zephyr.js
$ make update
Makefile:50: *** ZEPHYR_BASE not set. Source deps/zephyr/zephyr-env.sh.  Stop.
```
Branch | Build ID | Date | Result
---|---|---|---
master | 3f8eb85| Feb 13, 2017 | Fail

**Comment by grgustaf** [comment]
Fixed by @jprestwo in #720.

**Comment by haoxli** [comment]
Verified on commit 01aee7a93c7f4e739b562e907cbeb381049284cd, close it.
